cmd/statediff/car.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	bs "github.com/filecoin-project/lotus/lib/blockstore"
+	"github.com/ipld/go-car"
+	"github.com/ipfs/go-cid"
+	"github.com/urfave/cli/v2"
+
+	"github.com/filecoin-project/statediff"
+)
+
+var carFlags struct {
+	file string
+	preCid string
+	postCid string
+}
+
+var carCmd = &cli.Command{
+	Name:        "car",
+	Description: "Examine the state delta from a CAR",
+	Action:      runCarCmd,
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:        "file",
+			Usage:       "car store of cids",
+			Destination: &carFlags.file,
+			Required: true,
+		},
+		&cli.StringFlag{
+			Name:        "preCid",
+			Usage:       "initial state root",
+			Destination: &carFlags.preCid,
+			Required: true,
+		},
+		&cli.StringFlag{
+			Name:        "postCid",
+			Usage:       "final state root",
+			Destination: &carFlags.postCid,
+			Required: true,
+		},
```

Comment:
I would just make these positional arguments. The flags seem to add more noise than value. Could also use a range notation like Git, but I don't think it's worth it.